maquetado/css/styles.css

```diff
@@ -0,0 +1,43 @@
+.container {
+  display: flex;
+  flex-flow: column wrap;
+  justify-content: center;
+}
+
+.principal {
+  display: flex;
+  flex-flow: row wrap;
+  justify-content: center;
+  margin: 40px 0 0 0
```

Comment:
you shouldn't use margin here, you shoud use a padding in the top and bottom. try:
```
padding: 40px 0;
```